students/saleta_sebastian/lesson_09_reading_and_writing_files/regex_search.py

```diff
@@ -0,0 +1,25 @@
+import os
+import re
+
+
+def find_word_in_all_files(sentence):
+
+    for file in os.listdir(os.getcwd()):
+        if file.endswith(".txt"):
+            with open(file) as reader:
+                line_index = 0
+                for line in reader:
+                    line_index += 1
+                    if re.search(r'{}'.format(sentence), line):
+                        print('file : {}'.format(file))
+                        print('line {}: {}'.format(line_index, line))
+                    else:
+                        print('Cannot find more sentences in files')
```

Comment:
Is it printed too many times?

Reply:
I think it's ok and output is clearly for user. For example: 
python3 regex_search.py 
NOUN
file : story2.txt
line 1: The ADJECTIVE panda walked to the NOUN and then VERB. A nearby NOUN was unaffected by these events.

file : story.txt
line 1: The ADJECTIVE panda walked to the NOUN and then VERB. A nearby NOUN was unaffected by these events.